Author: Matthew Johnson <mjj29@debian.org>
Description: This using sectoid/Feelings instead of defy as the song for the credits. I'd
 like to ignore errors / disable music there, since we can't guarantee that
 even Feelings will be available, but I can't figure out how to catch the
 exception this bit of code is throwing.
Bug-Debian: http://bugs.debian.org/526399
Last-Update: 2009-05-02
--- a/src/Credits.py
+++ b/src/Credits.py
@@ -91,8 +91,8 @@
     self.engine      = engine
     self.time        = 0.0
     self.offset      = 1.0
-    self.songLoader  = self.engine.resource.load(self, "song", lambda: Song.loadSong(self.engine, "defy", playbackOnly = True),
-                                                 onLoad = self.songLoaded)
+    self.songLoader  = self.engine.resource.load(self, "song", lambda: Song.loadSong(self.engine, "sectoid/Feelings", playbackOnly = True),
+                                                    onLoad = self.songLoaded)
     self.engine.loadSvgDrawing(self, "background1", "editor.svg")
     self.engine.loadSvgDrawing(self, "background2", "keyboard.svg")
     self.engine.loadSvgDrawing(self, "background3", "cassette.svg")
